### DISCUSSION TITLE
2.2.11版本启动失败

### DISCUSSION THREAD

**Discussion by ruixiu** [milvus-io/milvus#25219] [Q&A and General discussion]
![03fead18099315156e591fbd1fe6c36](https://github.com/milvus-io/milvus/assets/9940354/61983426-fad2-47db-9a34-66adabe941e9)

**Comment by ruixiu** [comment]
2.2.10版本可以正常启动，把其中的2.2.10改成2.2.11就出现了上面的问题。docker-compose.yml内容如下：
version: '3.5'

services:
  etcd:
    container_name: milvus-etcd
    image: quay.io/coreos/etcd:v3.5.5
    restart: always
    environment:
      - ETCD_AUTO_COMPACTION_MODE=revision
      - ETCD_AUTO_COMPACTION_RETENTION=1000
      - ETCD_QUOTA_BACKEND_BYTES=4294967296
      - ETCD_SNAPSHOT_COUNT=50000
    volumes:
      - /workspace/data/milvus-yanshi/etcd:/etcd
    command: etcd -advertise-client-urls=http://127.0.0.1:2379 -listen-client-urls http://0.0.0.0:2379 --data-dir /etcd

  minio:
    container_name: milvus-minio
    image: minio/minio:RELEASE.2023-03-20T20-16-18Z
    restart: always
    environment:
      MINIO_ACCESS_KEY: minioadmin
      MINIO_SECRET_KEY: minioadmin
    volumes:
      - /workspace/data/milvus-yanshi/minio:/minio_data
    command: minio server /minio_data
    healthcheck:
      test: ["CMD", "curl", "-f", "http://localhost:9000/minio/health/live"]
      interval: 30s
      timeout: 20s
      retries: 3

  standalone:
    container_name: milvus-standalone
    image: milvusdb/milvus:v2.2.11
    restart: always
    command: ["milvus", "run", "standalone"]
    environment:
      ETCD_ENDPOINTS: etcd:2379
      MINIO_ADDRESS: minio:9000
    volumes:
      - /workspace/data/milvus-yanshi/milvus:/var/lib/milvus
    ports:
      - "19530:19530"
      - "9091:9091"
    depends_on:
      - "etcd"
      - "minio"
    
  attu:
    container_name: milvus-attu
    image: zilliz/attu:latest
    restart: always
    environment:
      - HOST_URL=http://192.168.40.130:8000
      - MILVUS_URL=192.168.40.130:19530
    ports:
      - "8000:3000"
      
networks:
  default:
    name: milvus
    ipam:
      config:
        - subnet: 172.19.0.0/16

**Comment by ruixiu** [comment]
![4262107e00df486e3e33928ef34ccae](https://github.com/milvus-io/milvus/assets/9940354/c1fc45fe-042a-4ce8-b9f6-8d80f9245150)

**Comment by ruixiu** [comment]
2.2.11之前的版本可能是代码bug，设置了两次optsKV.SetNumLevels(3)，但是没有设置optsStore.SetNumLevels(3)，所以optsStore的NumLevels的值默认是7，但是更新到2.2.11之后，变成了3，导致原数据的level比配置的值要大，所以会出现db has more levels than options.num_levels的错误，但是不知道如何处理

**Comment by yhmo** [comment]
已确认2.2.11有兼容问题，该版本已撤回
